documentation/book/security.adoc

```diff
@@ -14,8 +17,9 @@ All component certificates are signed by a Certification Authority (CA) called _
 Another CA is used for authentication of Kafka clients connecting to Kafka brokers.
 This CA is called _clients CA_.
 The CAs themselves use self-signed certificates.
+Finally, the Topic Operator has its own private and public key for encrypting communication with Kafka / Zookeeper.
```

Comment:
This seems a bit out of place. The previous sentences speak about CAs, so the _Finally_ IMHO doesn't fit in. 

Also, on the beginning of this paragraph we say _Each component needs its own private and public keys in order to support encryption._ ... that IMHO covers this part as well and we can simply delete the whole sentence, or?

Reply:
Agree